test/testdrive/webhook.td

```diff
@@ -288,7 +288,7 @@ $ webhook-append name=webhook_text_headers_block_list
 foo
 
 > SELECT body, headers::text FROM webhook_text_headers_block_list
-foo "{content-length=>3,host=>materialized:6876}"
+foo "{content-length=>3,host=>environmentd:6876}"
```

Comment:
@ParkMyCar: This change fixes the Nightly build but breaks https://buildkite.com/materialize/tests/builds/62955#018a4ffb-1d98-4b50-ab68-2d8a72fedea8. How can we influence this value?